### PR TITLE
feat: add custom button example

### PR DESCRIPTION
**Pull request by pinussilvestrus**
This adds an example of how to create a custom button component with a custom behavior on click. It also show cases how to add FEEL support for a property (`message`).

This uses not-yet-released hooks from `form-js-viewer`.

Kapture 2023-11-28 at 13 23 03

### CODE REVIEW

File: custom-button/app/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import './style.css';
+
+import schema from './form.json';
+
+new FormPlayground({
+  container: document.querySelector('#form'),
+  schema: schema,
+  data: {},
+  additionalModules: [
+    RenderExtension
```
Comment (Skaiir) [inline custom-button/app/index.js:19]: Might be worth prefixing Component Or CustomButton here, just for people to make the connection easier. I feel like when I see RenderExtension it's too generic and I glance over it a little, despite it being a critical part of the code.

Same for properties panel, but less important.

**Review by Skaiir — APPROVED** [review]
Feel free to merge or address the one thing. Thanks for this it'll help everybody :D

**Comment by Skaiir** [comment]
Ah whoops right automerge :trollface:

**Comment by pinussilvestrus** [comment]
Always progress 😄 

Adressed the small naming issue via https://github.com/bpmn-io/form-js-examples/commit/d83ad315b177449b48c74c5c9972eebc41d8675e 👍

**Pull request merged**